src/osmosis.go

```diff
@@ -59,7 +61,27 @@ type Pool struct {
 
 func getOsmosisPriceAndRoute(tokenInDenom, tokenOutDenom string, tokenInAmount int64) (float64, []poolmanagertypes.SwapAmountInSplitRoute, error) {
 	url := fmt.Sprintf("%s?tokenIn=%d%s&tokenOutDenom=%s&humanDenoms=false", osmosisQuoteAPI, tokenInAmount, tokenInDenom, tokenOutDenom)
-	resp, err := http.Get(url)
+
+	// Define the headers
```

Comment:
Can we make it so that we try to get the SQS OSMOSIS API KEY from env, if doesnt exist just to a get without the sqs api key? (for those who does not have access to sqs api key but still wants to use the arb bot)